wandb/sdk/wandb_artifacts.py

```diff
@@ -196,6 +203,12 @@ def add_manifest_file(log_phy_path):
         termlog("Done. %.1fs" % (time.time() - start_time), prefix=False)
 
     def add_reference(self, uri, name=None, checksum=True, max_objects=None):
+        if isinstance(uri, object):
```

Comment:
maybe some ' and ' instead of if chaining.

Reply:
👍

Reply:
incoming next commit